src/main/java/org/springframework/integration/aws/config/xml/AmazonWSParserUtils.java

```diff
@@ -48,11 +57,10 @@ private AmazonWSParserUtils() {
 	 * Registers the {@link AWSCredentials} bean with the current ApplicationContext if
 	 * accessKey and secretKey is given, if the credentials-ref is given, the given value
 	 * is returned.
-	 *
 	 * @param element
 	 * @param parserContext
 	 */
-	public static String getAmazonWSCredentials(Element element,ParserContext parserContext) {
+	public static String getAmazonWSCredentials(Element element, ParserContext parserContext) {
```

Comment:
I would prefer to use the Spring Cloud AWS Credentials provider setup. As that arrangement support to use different type of credentials provider like simple (used here), instance profile and also a dedicated profile file. The "chain-of-reponsibility" implementation can query multiple ones until one of them returns actually the credentials.